lua/neogit/lib/git/config.lua

```diff
@@ -147,4 +147,16 @@ function M.unset(key)
   cli.config.unset(key).call_sync()
 end
 
+function M.get_comment_char()
+  local char = M.get("core.commentChar").value
+  if char == "" then
+    char = M.get_global("core.commentChar").value
+    if char == "" then
+      char = "#"
+    end
+  end
+
+  return char
+end
```

Comment:
I don't really wanna have bespoke value getters in this library, but we can simplify how it's written in the callers to obviate the need for a function:

```lua
local comment_char = git.config.get("core.commentChar"):read() or git.config.get_global("core.commentChar"):read() or "#"
```

Reply:
That's exactly what I was asking about. Will change code in a moment.